### PR TITLE
Add wording for file not found

### PR DESCRIPTION
**Pull request by tevincent**

### CODE REVIEW

File: src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -30,6 +30,7 @@
     <string name="connectionError">Problème de connexion internet</string>
     <string name="errorDownload">Erreur lors du téléchargement</string>
     <string name="errorDownloadInsufficientSpace">Téléchargement impossible : Espace insuffisant</string>
+    <string name="errorFileNotFound">Une erreur est survenu lors de l’ouverture de ce fichier</string>
```
Comment (LunarX) [inline src/main/res/values-fr/strings.xml:33]: Is it not "est survenu**e**"?

File: src/main/res/values/strings.xml [hunk 2]
```diff
@@ -32,6 +32,7 @@
     <string name="connectionError">Internet connection problem</string>
     <string name="errorDownload">Download error</string>
     <string name="errorDownloadInsufficientSpace">Cannot download : Insufficient space</string>
+    <string name="errorFileNotFound">An error occurred when opening this file</string>
```
Comment (LunarX) [inline src/main/res/values/strings.xml:35]: Wouldn't "An error **has** occurred" show that the error **just** occurred right now and therefore be a bit more precise?

**Review by LunarX — CHANGES_REQUESTED** [review]

File: src/main/res/values/strings.xml (hunk 2 above)
Comment (FabianDevel) [reply src/main/res/values/strings.xml:35]: My take would be "An error has occured while opening this file"

Comment (tevincent) [reply src/main/res/values/strings.xml]: occurred?

**Review by LunarX — APPROVED** [review]

**Pull request merged**